File: C to Java/Assignments/Assignment 6/Assignment 4 with Function/q2.c
/* Print prime numbers in the given range 1 to n.
*/

void printprime(int n) {
	for(int i=1 ; i <= n ; i++) {
		int no = i ;
		int prime = 1 ;
		
		if(no <= 1 ) {
			prime = 0 ;
		}
		for(int j = 2 ; j <= i /2 ; j++){
			if(no % j == 0) {
				prime = 0 ;
				break ;
			}
		}
		
		if(prime){
			printf("%d   ", i);
		}
		
	}
}

void main() {
	
	int n ;
	printf("enter number : ");
	scanf("%d" , &n);
	
	printprime(n);
	
}

File: C to Java/Assignments/Assignment 6/Assignment 1 with Function/q2.java
/* Print armstrong numbers in the given range 1 to n.
*/

import java.util.Scanner;

public class q2 {
	
	

	

	static int count(int n){
		int count = 0 ;
		while(n > 0) {
			n = n / 10;
			count ++ ;
		}
	
		return count ;
	}


	static int power(int n , int count){
		int power=1;
		for(int i=1; i <= count; i++){
			power = n * power ;
		} 
	
		return power ;
	}


	static void armstrong(int n) {
		for(int i = 1 ; i <= n ; i++){
		
			int sum = 0 ;
			int temp = i ;
		
			int cnt = count(i);
				while(temp > 0 ) {
					int	digit = temp % 10 ;
					sum = sum + power(digit , cnt);
					temp = temp / 10 ;
				}
		
			if(i == sum){
				System.out.printf("%d ", i);
			}
		}
	}


	public static void main(String args[]) {

		Scanner sc = new Scanner(System.in);

		int n ;
		System.out.printf("enter range n : ");
		n = sc.nextInt();

		armstrong(n);
	
	}


}